src/lib/viewers/image/ImageViewer.js

```diff
@@ -531,6 +541,10 @@ class ImageViewer extends ImageBaseViewer {
         if (this.areNewAnnotationsEnabled()) {
             this.annotator.addListener('annotations_create', this.handleAnnotationCreateEvent);
         }
+
+        if (this.cache.get(IMAGE_FTUX_CURSOR_DISABLED_KEY)) {
+            this.annotator.emit('annotations_image_explicit_create_toggled');
```

Comment:
This is required because we want to make sure that the PopupCursor is hidden properly on a refresh.